source/CIS/cis-1.rst

```diff
@@ -616,28 +662,33 @@ The specification only has a ``transfer`` smart contract function which takes li
 This will result in lower energy cost compared to multiple contract calls and only introduces a small overhead for single transfers.
 The reason for not also including a single transfer function is to have smaller smart contract modules, which in turn leads to saving cost on every function call.
 
+No explicit authentication
```

Comment:
Should this be authorization instead?

I suppose we also don't mandate any authentication other than that provided by the chain (so sender/owner are authenticated).